### PR TITLE
Veres One 'capybara' release

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Work in progress PR, compatible with the Veres One `capybara` release.

Ledgers Supported:
1. Veres One (capybara, dev mode and soon testnet)

Features:
* [x] Local storage of private keys, metadata, and copies of DID Documents on filesystem.
* [x] `generate()` did (gets put into the 'pending' storage)
* [x] `register()` did on ledger
* [x]  Fetch DID from ledger (`./did info <did>`)
* [x]  List DIDs (local and registered) (`./did list`)
* [x]  Delete DID
* [x] Rotate key - ready, see PR https://github.com/digitalbazaar/did-cli/pull/27
* [ ] Add new key
* [ ] Revoke key

### CODE REVIEW

**Comment by mattcollier** [comment]
@dmitrizagidulin we will not be doing services in capybara

**Comment by dmitrizagidulin** [comment]
@mattcollier ahaaa, ok, removing.

File: README.md [hunk 1]
```diff
@@ -34,10 +33,7 @@ To retrieve the freshly generated DID:
 
 ### Requirements
 
-* Linux
-  * g++ to build native binaries
-* Mac OS X
-  * X Code 9 to build native binaries
+* Linux or Mac OS X
```
Comment (davidlehn) [inline README.md:36]: The previous Linux/MacOS list was there to mention the build tools.  If there are no native deps then code should work on any Node.js platform and no need to list OSes in particular.
```suggestion
```

Comment (dmitrizagidulin) [reply README.md:36]: I hear you. I'll change the wording to 'Only tested on Linux and Mac OS X'. :)

Comment (davidlehn) [reply README.md:36]: Just say nothing.  It not working on any Node.js platform is a bug somewhere.

**Comment by davidlehn** [comment]
An attempt was made to rename this branch to `capybara-rc`, which closed this PR. I'm not sure how to change the PR source branch. A new PR may be needed. Reopened this for now due to PR text and comments.

**Pull request closed without merging**